CHANGELOG.asciidoc

```diff
@@ -29,6 +29,24 @@ Notes:
 === Node.js Agent version 3.x
 
 
+==== Unreleased
+
+[float]
+===== Breaking changes
+
+[float]
+===== Features
+
+[float]
+===== Bug fixes
+
+* Guard against a negative value of `metricsInterval`, which can lead to
```

Comment:
Is it worth calling out in the CHANGELOG that the configuration handling for 'abortedErrorThreshold', 'apiRequestTime' are also changed/fixed such that negative or null/undefined values now result in the default values being used?

Reply:
Done in commit f9e992a842607f931d708e75331aa76dfe738c86